README.md

```diff
@@ -1,2 +1,57 @@
-# go-n2k
-Golang NMEA 2000 parsing library
+## boatkit-io/n2k
+
+boatkit-io/n2k comprises a package (and associated tools) that provides a subscription model enabling registered clients to receive strongly typed [go](https://go.dev) data structures generated from a stream of canbus-based NMEA 2000 messages.
```

Comment:
NMEA 2000 messages that may come from canbus or data other sources?

Reply:
fixed